src/system.h

```diff
@@ -91,6 +91,11 @@
 #  define SUPPORT_MOUSE
 #  define SUPPORT_JOYSTICK
 #  define SUPPORT_JOYSTICK_AXIS
+#elif defined(__PS4__)
+#  define SUPPORT_ZOOM
+#  define SUPPORT_JOYSTICK
+#  define SUPPORT_JOYSTICK_AXIS
+#  define SUPPORT_FILE_BROWSER
```

Comment:
The `SUPPORT_FILE_BROWSER` is for opening file open dialog in the soundfont and the font settings. Can you remove the define as this will only work on desktop platforms (or does the PS4 support anything like this?)